views/js/qtiCreator/widgets/interactions/graphicInteraction/Widget.js

```diff
@@ -114,7 +114,7 @@ define([
          * @private
          * @param {Object} choice - the QTI choice
          */
-        _currentChoices : function(choice){
+        _currentChoices : (choice) => {
```

Comment:
I'm not sure that's the right way to bind the context to the function.
Consider this:
1. Revert this function back
2. Change `_.forEach(this.element.getChoices(), this._currentChoices, this);` above to `_.forEach(this.element.getChoices(), _.bind(this._currentChoices, this));`

Reply:
You have right, fixed